docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -772,6 +772,10 @@ The output of the query has the following history columns:
 * - `version`
   - `BIGINT`
   - The version of the table corresponding to the operation
+* - `creation_time`
```

Comment:
in_commit_timestamp

Reply:
You have `in_commit_timestamp` in the code everywhere and that comment was about renaming existing `timestamp` column

Reply:
Oh, I see.